telescope/selector_test.py

```diff
@@ -126,6 +137,7 @@ def testValidInput_v1dot1_Simple(self):
                                                  {'db_snapshots': ['2014-08-04']}))
     selector_expected.site = 'lga02'
     selector_expected.client_provider = 'comcast'
+    selector_expected.client_country = 'us'
```

Comment:
I think we need several more test cases here because the selector format is significantly more complex now that we have fields that can be either / or / both. What are the rules exactly with sites, providers, and countries? Is the user allowed to specify just one of sites, providers, and countries?